src/WinGet.RestSource.Utils/Constants/ApiConstants.cs

```diff
@@ -56,6 +56,7 @@ public class ApiConstants
         {
             "1.0.0",
             "1.1.0",
+            "1.4.0",
         };

```

Comment:
So if someone sets 1.2.0 we will fail?

Reply:
Yes. There is no 1.2.0.

Reply:
If I remember correctly, the check validates that the ManifestVersion attribute match any of these otherwise it fails. There are tons of `ManifestVersion: 1.2.0` in winget-pkgs so unless I'm wrong the validation is going to fail.

Reply:
This is for the API version, not necessarally for the manifest version. When I spoke with @yao-msft he said that we were skipping 1.2, and his PR for schema is 1.4. We don't have a 1.2 or a 1.3.

Reply:
To clarify, these two version numbers don't necessarily have to be in sync. The YAML manifest versions are distinct from the REST versions. So far, however, we have keep them in sync.

Reply:
ManifestVersion is different concept from rest api version, the rest api version 1.4 consists of changes from ManifestVersion 1.2 and 1.4. We decide to name it v1.4 because from now on, all versions should match winget versions. For example, in ManifestVersion, we skipped v1.3

I don't know the internal implementation of rest source well but If we are not directly converting the ManifestVersion to rest api version, it's fine